de/fitting.py

```diff
@@ -52,7 +52,7 @@ def runOptim(data, niter=2000, disp=0):
 
     U = np.copy(data)
     np.fill_diagonal(U, 0.0)
-    cost_grad = jit(grad(cost, argnums=0), static_argnums=(3,))
```

Comment:
Is this change necessary?

Reply:
I guess so. Since in the last merged PR we no longer have the `linear` argument in the fitting (as it is now handled in another function with Lasso) having the `static_argnums` gives an error when it runs.

Reply:
I'm honestly not sure what this does, I don't think it was my edit but I just removed the bellmanford package!

Reply:
Ok, push your edit then I'll approve.